.ci/install_qemu.sh

```diff
@@ -201,29 +78,38 @@ main() {
 		esac
 	done
 
+	export qemu_type
 	case "${qemu_type}" in
-		tdx)
-			CURRENT_QEMU_VERSION=$(get_version "assets.hypervisor.qemu.tdx.tag")
-			QEMU_REPO_URL=$(get_version "assets.hypervisor.qemu.tdx.url")
-			qemu_latest_build_url="${jenkins_url}/job/kata-containers-2.0-qemu-tdx-$(uname -m)/${cached_artifacts_path}"
-			;;
 		vanilla)
+			qemu_type="qemu"
+			;;
+		*)
+			die_unsupported_qemu_type "$qemu_type"
```

Comment:
Same comment here: I don't see this call defined or used anywhere else.

Reply:
Nice catch!  I missed adding those two that were present as part of the CCv0 branch but not here.